src/rules/auto-import-tokens.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-present Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const merge = require('lodash/merge');
+const snakeCase = require('lodash/snakeCase');
+const { props: TOKENS } = require('bpk-tokens/tokens/base.raw.json');
+
+const BASE_CONFIG = {
+  typeof: false,
+  platform: 'web',
+  tokensPackage: {
+    web: 'bpk-tokens/tokens/base.es6',
+    native: 'bpk-tokens/tokens/base.react.native'
+  }
+};
+
+const isToken = value => !!TOKENS[snakeCase(value).toUpperCase()];
+
+const hasTypeOfOperator = ({ parent }) =>
+  parent && parent.type === 'UnaryExpression' && parent.operator === 'typeof';
+
+module.exports = context => {
+  const options = merge(BASE_CONFIG, context.options[0] || {});
+  const tokenPkg = options.tokensPackage[options.platform];
+
+  return {
+    meta: {
+      fixable: 'code',
+      schema: [
+        {
+          type: 'object',
+          properties: {
+            typeof: {
+              type: 'boolean'
+            },
+            platform: {
+              type: 'string'
+            },
+            tokensPackage: {
+              type: 'object',
+              properties: {
+                web: {
+                  type: 'string'
+                },
+
+                native: {
+                  type: 'string'
+                }
+              },
+              additionalProperties: false
+            }
+          },
+          additionalProperties: false
+        }
+      ]
+    },
+
+    'Property:exit': (/* node */) => {
```

Comment:
`exit` means it runs when going up the tree. Basically it means it runs last, which we need because this needs to execute after the `use-backpack` rule.